website/src/contribute/index.md

```diff
@@ -77,13 +70,24 @@ To contribute code, you need
 
 ## Submitting your first code change
```

Comment:
How about "Contributing Code" - since there are several other steps before we are ready to submit something?

Reply:
Great - done.